docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3.3'
```

Comment:
How does this differ from the one in https://github.com/zalando-incubator/remora/blob/master/docker-compose/docker-compose.yml if we need updates to the docker-compose file please keep them all in the one file unless there's a different use case being tested.

Reply:
Forgot that existed, updated